extension/intents/notes/notes.toml

```diff
@@ -16,6 +16,15 @@ match = """
 [[notes.addLink.example]]
 phrase = "Make note of this page"
 
+[notes.paste]
+description = "Focus the tab where you want to paste"
```

Comment:
This won't focus the tab, a better description would just be "Pastes the clipboard into your note"

Reply:
Yup I've made the changes.
Is there a naming convention for the descriptions and phrases? Wanted to learn about that so I don't repeat these mistakes again.

Reply:
The description isn't particularly important, it's just there for developers. But it's always good to be accurate ;)

Examples should usually be easy for the user to understand – so even if you match a short form, you might put in a longer form of the phrase in the example so that it's obvious what it will do. There's no extra information given with examples, so they have to be self-explanatory.

For phrase matching it's just a matter of trying to think up different ways someone might phrase the command. It's more art than science at this point!

Reply:
Thanks @ianb 
I'll keep this in mind while working :)